app/i18n/formatters.ts

```diff
@@ -6,11 +6,14 @@ export const initFormatters: FormattersInitializer<Locales, Formatters> = (
   locale: Locales,
 ) => {
   const formatters: Formatters = {
-    sats: (value: number): string => {
+    sats: (value: unknown): unknown => {
```

Comment:
Why is this unknown?

Reply:
this is what the library i18n expects AFAIU